static/app/components/onboarding/gettingStartedDoc/types.ts

```diff
@@ -95,6 +97,16 @@ export interface Docs<PlatformOptions extends BasePlatformOptions = BasePlatform
   customMetricsOnboarding?: OnboardingConfig<PlatformOptions>;
   feedbackOnboardingCrashApi?: OnboardingConfig<PlatformOptions>;
   feedbackOnboardingNpm?: OnboardingConfig<PlatformOptions>;
+  onProductSelectionChange?: (params: {
```

Comment:
Would move this into the `OnboardingConfig` type. Change handlers that we implement for the general onboarding will most likely not be relevant / wanted in e.g. a sidebar doc.